.NET/Microsoft.Recognizers.Text.sln

```diff
@@ -803,6 +803,11 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Turkish", "Turkish", "{2F41
 		..\Specs\DateTime\Turkish\TimePeriodParser.json = ..\Specs\DateTime\Turkish\TimePeriodParser.json
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Swedish", "Swedish", "{2AA6937E-36C0-4E6B-B536-AA0E08D30261}"
```

Comment:
Nitpick: Can you manually edit the file so this Swedish spec additional is near the other ones (clustered by culture)?